File: v2/csharp/src/main/csharp/io/swagger/Model/Articles.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Articles {
    

    
    public List<Article> Articles { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Articles {\n");
      
      sb.Append("  Articles: ").Append(Articles).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Articles.java
package vestorly.client.model;

import java.util.*;
import vestorly.client.model.Article;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Articles  {
  
  private List<Article> articles = new ArrayList<Article>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("articles")
  public List<Article> getArticles() {
    return articles;
  }
  public void setArticles(List<Article> articles) {
    this.articles = articles;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Articles {\n");
    
    sb.append("  articles: ").append(articles).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
